src/pages/RegisterStaff.jsx

```diff
@@ -0,0 +1,82 @@
+import { RegisterStaffForm } from '@/forms/staff/forms';
+import { registerStaffSchema } from '@/forms/staff/schemas';
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from '@/components/ui/card';
+import { registerStaff } from '@/services/staff';
+import { useState } from 'react';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useForm } from 'react-hook-form';
+import { useNavigate } from 'react-router';
+
+export function RegisterStaff() {
+    
+  return (
+    <div className="flex min-h-screen items-center justify-center">
+      {/* Right side - Register Staff Form */}
+      <div className="w-full lg:w-2/3 flex items-center justify-center p-8">
+        <RegisterStaffCard />
+      </div>
+    </div>
```

Comment:
There is no left side. Please center

Reply:
Resolved in https://github.com/FIS2425/frontend/pull/30/commits/8c4521b8378997b038e5f302dddf72c82ba3b2b8